src/ObjectUtils/PdbFileLlvm.cpp

```diff
@@ -228,6 +228,18 @@ ErrorMessageOr<std::unique_ptr<PdbFile>> PdbFileLlvm::CreatePdbFile(
     return ErrorMessage(absl::StrFormat("Unable to load PDB file %s with error: %s",
                                         file_path.string(), llvm::toString(std::move(error))));
   }
+  auto* native_session = dynamic_cast<llvm::pdb::NativeSession*>(session.get());
```

Comment:
So all the added code here is to guarantee that `GetAge` will succeed? I think a comment would be beneficial.

We can also consider extracting a function `static bool PdbHasDbiStream(llvm::pdb::IPDBSession* session)`. This also saves you from creating the error string unless that function returns false.

Reply:
Done.